crosswalks/MODS.csv

```diff
@@ -0,0 +1,68 @@
+﻿Property,MODS
+codeRepository,relatedItem/[@type='Host']
+programmingLanguage,
+runtimePlatform,
+targetProduct,
+applicationCategory,
+applicationSubCategory,
+downloadUrl,location/url[@displayName='Download URL']
+fileSize,physicalDescription/extent
+installUrl,location/url[@displayName='Install URL']
+memoryRequirements,
+operatingSystem,
+permissions,
+processorRequirements,
+releaseNotes,description
+softwareHelp,relatedItem
+softwareRequirements,
+softwareVersion,originInfo/edition
+storageRequirements,
+supportingData,relatedItem[@type='references']
+author,name/role/roleTerm[@type='code']/aut
+citation,relatedItem[@otherType='references']
+contributor,name/role/roleTerm[@type='code']/ctb
+copyrightHolder,accessCondition/rights.holder/name
+copyrightYear,accessCondition/creation/year.creation
+dateCreated,originInfo/dateCreated
+dateModified,originInfo/dateModified
+datePublished,originInfo/dateIssued
+editor,name/role/roleTerm[@type='code']/edt
+encoding,
+fileFormat,typeOfResource
+funder,name/role/roleTerm[@type='code']/fnd
+keywords,subject
+license,accessCondition[@type='use and reproduction']
+producer,name/role/roleTerm[@type='code']/pro
+provider,name/role/roleTerm[@type='code']/prv

```

Comment:
```suggestion
author,name/role/roleTerm[@type='code']/[.='aut']
citation,relatedItem[@otherType='references']
contributor,name/role/roleTerm[@type='code']/[.='ctb']
copyrightHolder,accessCondition/rights.holder/name
copyrightYear,accessCondition/creation/year.creation
dateCreated,originInfo/dateCreated
dateModified,originInfo/dateModified
datePublished,originInfo/dateIssued
editor,name/role/roleTerm[@type='code']/[.='edt']
encoding,
fileFormat,typeOfResource
funder,name/role/roleTerm[@type='code']/[.='fnd']
keywords,subject
license,accessCondition[@type='use and reproduction']
producer,name/role/roleTerm[@type='code']/[.='pro']
provider,name/role/roleTerm[@type='code']/[.='prv']
```

Let's use the XPath syntax for text; because they aren't element in [the example I found](https://www.loc.gov/standards/mods/userguide/introduction.html#implementation_notes)
```